pkg/storage/store.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // Store performs database operations for EASi
 type Store struct {
-	db        *sqlx.DB
+	DB        *sqlx.DB
```

Comment:
I didn't do a full look through the changes, but I wanted to check on this. Is this so you can use the DB as a named preparer? Is there any other reason the DB is exposed?

Have you tried implementing  the new function of named preparer on the store instead? That would keep the DB separated, and still provide the functionality you are looking for